gameinfo.conf.template

```diff
@@ -0,0 +1,7 @@
+NAME               GameName
```

Comment:
I don't think anything prevent to use a real-life example like Unvanquished default strings in the template.

Reply:
Same for the default values in the code if undefined.

Note that in the future we would be able to make the engine being able to load some built-in asset and display an error (like Darkplaces does) so the template and the hardcoded version would reference that dummy game that does nothing at all.